modules/security-context-constraints-about.adoc

```diff
@@ -156,6 +153,13 @@ The restricted SCC is the most restrictive of the SCCs that ship by default with
 * `seccompProfile` is set to `runtime/default` by default.
 * `allowPrivilegeEscalation` must be unset or set to `false` in security contexts.
 
+This is the most restrictive SCC provided by a new installation and will be used by default for authenticated users.
+
+[NOTE]
+====
+The `restricted-v2` SCC is the most restrictive of the SCCs that ship by default with the system. However, you can create a custom SCC that is even more restrictive. For example, you can create an SCC that restricts `readOnlyRootFS` to `true`.
```

Comment:
Consider revising `ship` -> `ships` or `is shipped`.

Reply:
Thanks @sayjadha! I checked ISG and they recommend "is included" instead of "ship", so I updated to that.